proxy/http/HttpSM.cc

```diff
@@ -494,6 +494,12 @@ HttpSM::attach_client_session(ProxyTransaction *client_vc, IOBufferReader *buffe
     debug_on = true;
   }
 
+  ink_release_assert(ua_txn->get_proxy_ssn());
```

Comment:
I think these should be debug asserts.

Reply:
I can change it to that. Do debug asserts execute in Autests and regression tests?

Reply:
Yes.